easybuild/easyconfigs/n/netCDF/netCDF-4.7.1-gompi-2019a.eb

```diff
@@ -0,0 +1,34 @@
+name = 'netCDF'
+version = "4.7.1"
+
+homepage = 'http://www.unidata.ucar.edu/software/netcdf/'
+description = """NetCDF (network Common Data Form) is a set of software libraries 
+ and machine-independent data formats that support the creation, access, and sharing of array-oriented 
+ scientific data."""
+
+toolchain = {'name': 'gompi', 'version': '2019a'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+source_urls = ['https://github.com/Unidata/netcdf-c/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['583e6b89c57037293fc3878c9181bb89151da8c6015ecea404dd426fea219b2c']
+
+builddependencies = [
+    ('Autotools', '20180311'),
+    ('CMake', '3.13.4'),
```

Comment:
@henkela Why not use CMake 3.15.3, for which an existing easyconfig that can be used to resolve this build dep is already available (`CMake-3.15.3-GCCcore-8.3.0.eb`)?